app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,74 @@
+<%= form_with(model: @recipe, url: recipes_path, local: true) do |form| %>
+  <div class = "mt-4 p-2">
+    <%= select_tag(:public, options_for_select([['Select Status', 'Status'], ['Public', 'Public'], ['Private', 'Private']], 'Status'),
+      class: 'form-select')
+    %>
+    <% if @recipe.errors.any? %>
+        <p class="text-danger mb-0">
+          <%= @recipe.errors[:status].first %>
+        </p>
+    <% end %>
+  </div>
+
+	<div class = "mt-2 p-2">
+		<%= form.text_field :name,
+        autofocus: true, 
+        autocomplete: 'name', 
+        placeholder: 'New inventory name', 
+        class: 'form-control' 
+    %>
```

Comment:
- [ ] I suggest that the placeholder for this should be `New recipe name` instead of 'New inventory name' since this form is for creating recipes not inventory :blush: